doc.go

```diff
@@ -131,7 +131,9 @@ The request has a valid URL but the method (GET,PUT,POST,...) is not allowed.
 
 	406: Not Acceptable
 
-The request does not have or has an unknown Accept Header set for this operation.
+The request does not have or has an unknown Accept Header set for this operation. Note that a 406 might also occur
```

Comment:
The issued described is that on the server side you need to specify a .Consumes() clause even on calls with an empty body. So it would be something like 'Note that a 406 might also occur if no .Consumes("application/octet-stream") statement has been set on the endpoint'. I'll check whether application/octet-stream is an example like json, xml or required configuration for empty calls.